templates/docs/examples/patterns/code-snippet/code-snippet.html

```diff
@@ -0,0 +1,29 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Code Snippet{% endblock %}
+
+{% block content %}
+<div class="p-code-snippet">
+  <div class="p-code-snippet__header">
+    <h5 class="p-code-snippet__title">This is the code snippet title</h5>
+  </div>
+
+  <pre class="p-code-snippet__block"><code>something.test();</code></pre>
+
+  <div class="p-code-snippet__header">
+    <h5 class="p-code-snippet__title">Output</h5>
+  </div>
+
+  <pre class="p-code-snippet__block"><code>Test passed!</code></pre> 
+  
+  <div class="p-code-snippet__header">
```

Comment:
While it is possible to have multiple blocks I think the intended structure is max 2 (code + output), so we shouldn't put more to our examples.

If we want to show some separate use, it should be separate code snippet (in same example) or separate example.